articles/machine-learning/tutorial-azure-ml-in-a-day.md

```diff
@@ -215,6 +215,127 @@ You might need to select **Refresh** to see the new folder and script in your **
 
 :::image type="content" source="media/tutorial-azure-ml-in-a-day/refresh.png" alt-text="Screenshot shows the refresh icon.":::
 
+### [Optional] Enable Intel® Extension for Scikit-Learn optimizations for more performance on Intel hardware
+
+Want to speed up your scikit-learn scripts on Intel hardware? Try enabling [Intel® Extension for Scikit-Learn](https://www.intel.com/content/www/us/en/developer/tools/oneapi/scikit-learn.html) in your training script. Intel® Extension for Scikit-Learn is already installed in the Azure Machine Learning curated environment used in this tutorial, so no additional installation is needed.
+
```

Comment:
Are you sure that the extension is installed?  I just tried using your script and got this in the error log:
Traceback (most recent call last):
  File "main.py", line 6, in <module>
    from sklearnex import patch_sklearn
ModuleNotFoundError: No module named 'sklearnex'

Reply:
@sdgilley, this will be installed using changes in the AzureML notebook referenced here: https://github.com/Azure/azureml-examples/pull/2429

Reply:
Got it.  

We still can't complete this PR until the azureml-examples notebook is merged.  Could you perhaps close this for now, then reopen once the other notebook is available?  

I'll check with the other repo folks to get someone to look at your azureml-examples PR.

Reply:
I am working with Samantha Salgado from MSFT on this effort, she said she would connect with the azureml-examples PR. Maybe connect with her?